WcaOnRails/spec/requests/api_persons_spec.rb

```diff
@@ -53,4 +53,16 @@
       expect(json["personal_records"]["333"]["average"]["best"]).to eq 590
     end
   end
+
+  describe "GET #results" do
+    let!(:result) { FactoryBot.create :result, person: person }
+    it "renders properly" do
+      get api_v0_person_results_path(result)
```

Comment:
This was also good as `get api_v0_person_results_path(person)`, rails uses the `person` object to figure out the person id in the endpoint.